packages/blade/src/components/Dropdown/useDropdown.ts

```diff
@@ -234,6 +236,11 @@ const useDropdown = (): UseDropdownReturnValue => {
       setActiveIndex(-1);
     }
 
+    if (bottomSheetContext?.hasBottomSheet) {
+      setShouldIgnoreBlur(true);
+      return;
+    }
+
     if (shouldIgnoreBlur) {
```

Comment:
```suggestion
    if (shouldIgnoreBlur && !bottomSheetContext.hasBottomSheet) {
```

will this do the same thing?